pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.mypy]
+disallow_untyped_defs = true
+ignore_missing_imports = true
+no_implicit_optional = true
+show_error_codes = true
+files = ["src/pybreaker.py"]
+
+[tool.isort]
+profile = "black"
```

Comment:
I forgot this when I added `black` and `isort`.

If this is not set, they may mismatch.